vowpalwabbit/cb_explore_adf_squarecb.cc

```diff
@@ -73,14 +80,16 @@ struct cb_explore_adf_squarecb
 };
 
 cb_explore_adf_squarecb::cb_explore_adf_squarecb(
-    float gamma_scale, float gamma_exponent, bool elim, float c0, float min_cb_cost, float max_cb_cost)
+    float gamma_scale, float gamma_exponent, bool elim, float c0, float min_cb_cost, float max_cb_cost,
+    VW::version_struct model_file_version)
     : _counter(0)
     , _gamma_scale(gamma_scale)
     , _gamma_exponent(gamma_exponent)
     , _elim(elim)
     , _c0(c0)
     , _min_cb_cost(min_cb_cost)
     , _max_cb_cost(max_cb_cost)
+    , _model_file_version(model_file_version)
```

Comment:
Avoid the extra copy
```suggestion
    , _model_file_version(std::move(model_file_version))
```